packages/now-cli/test/integration.js

```diff
@@ -2916,3 +2916,19 @@ test('reject conflicting `vercel.json` and `now.json` files', async t => {
     formatOutput({ stderr, stdout })
   );
 });
+
+test('`vc --debug project ls` should output the projects listing', async t => {
+  const { exitCode, stderr, stdout } = await execa(
+    binaryPath,
+    [...defaultArgs, '--debug', 'project', 'ls'],
```

Comment:
I'm not sure if it will make a difference, but usually defaultArgs is at the end

Reply:
Good point, but I don't think it makes a difference in this case (the issue was happening when `--debug` came before `project`). This test was copied from the test above.